### DISCUSSION TITLE
What is "wip" in git stands for after the branch name?

### DISCUSSION THREAD

**Discussion by jasursadikov** [romkatv/powerlevel10k#2602] [General]
![image](https://github.com/romkatv/powerlevel10k/assets/14846427/c2221f90-966a-40c9-8e5f-ab5e5743d588)
What is this WIP thing?

**Comment by romkatv** [comment]
https://github.com/romkatv/powerlevel10k?tab=readme-ov-file#what-do-different-symbols-in-git-status-mean
